IrrlichtExtensions/BeautifulCheckBox.cpp

```diff
@@ -67,6 +69,8 @@ bool BeautifulCheckBox::OnEvent(const irr::SEvent& event){
 				if(Parent && isTrulyVisible()){
 					checked = !checked;
 					BeautifulGUIText::OnEvent(event);//important, otherwise e.g. parent aggregations don't scroll properly
+					irr::SEvent checkBoxEventCopy = checkBoxEvent;
+					Parent->OnEvent(checkBoxEventCopy);
```

Comment:
Should have already been done by IGUIElement::OnEvent which is called by IAggregatableGUIElement::OnEvent